src/uct/api/v2/uct_v2.h

```diff
@@ -93,7 +93,10 @@ enum uct_perf_attr_field {
     UCT_PERF_ATTR_FIELD_LATENCY            = UCS_BIT(9),
 
     /** Enable @ref uct_perf_attr_t::max_inflight_eps */
-    UCT_PERF_ATTR_FIELD_MAX_INFLIGHT_EPS   = UCS_BIT(10)
+    UCT_PERF_ATTR_FIELD_MAX_INFLIGHT_EPS   = UCS_BIT(10),
+
+    /** Enable @ref uct_perf_attr_t::remote_guid */
+    UCT_PERF_ATTR_FIELD_REMOTE_GUID        = UCS_BIT(11)
```

Comment:
instead of adding new field, change the remote sys_dev type to ucs_sys_device_uid_t